package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-video-renderer",
-  "version": "1.1.2",
```

Comment:
We broke the public api with **VideoStatus** type, hence it's major change hehe

Reply:
yeap 👍